install/operator/pkg/generator/assets/install/app.yml.tmpl

```diff
@@ -6,5 +6,5 @@ spec:
   addons:
   {{- range .EnabledAddons }}
     {{.}}:
-      enabled: "true"
+      enabled: true
```

Comment:
This may or may not be necessary depending on whether https://github.com/syndesisio/syndesis/pull/7259 gets merged first.